frontend/src/App.css

```diff
@@ -140,13 +140,15 @@ body {
   font-size: 1.6;
   color: var(--color-gray-700);
 
-  overflow: hidden;
+  overflow-x: hidden;
```

Comment:
![Screenshot 2024-05-31 110352](https://github.com/Satyam1923/Spring-Music-Player/assets/104068640/9e00fff0-1367-4b00-b9d8-c3c36521ffb7)
see to it that it doesn't go above the normal ui.

Reply:
here i have removed coverflow hidden since it wont let us scroll in y direction and added overflow-x hidden